modules/openapi-generator/src/main/resources/typescript/http/http.mustache

```diff
@@ -112,7 +112,7 @@ export class RequestContext {
     	return this.headers;
     }
     
-    public getBody(): string | FormData { 
+    public getBody(): string | FormData | undefined{
```

Comment:
Would it make sense to define a type for this? `type Body = string | FormData | undefined` or something similar?

Reply:
I'll have to think about that ;-)

Reply:
I have added a type `RequestBody`. `setBody` can also be used with it, so setting the body to `undefined` removes it.